### PR TITLE
upgrade of centauri pallet and set IBC gates #CU-8669mcw9g

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
For https://app.clickup.com/t/8669mcw9g

Upgraded Centauri to new version. 

Setup release gates for IBC pallet.

### CODE REVIEW

**Review by obsessed-cake — APPROVED** [review]

File: code/parachain/runtime/picasso/src/ibc.rs [hunk 1]
```diff
@@ -104,6 +105,28 @@ impl core::str::FromStr for MemoMessage {
...
+use pallet_ibc::ics20::Ics20RateLimiter;
+
+pub struct ConstantAny;
+
+impl Ics20RateLimiter for ConstantAny {
+	fn allow(
+		msg: &pallet_ibc::ics20::Ics20TransferMsg,
+		_flow_type: pallet_ibc::ics20::FlowType,
+	) -> Result<(), ()> {
+		// one DOT/PICA, so so for USDT not safe, but we do not yet do it
```
Comment (blasrodri) [inline code/parachain/runtime/picasso/src/ibc.rs:122]:
```suggestion
		// one DOT/PICA, so for USDT not safe, but we do not yet do it
```

File: code/parachain/runtime/picasso/src/ibc.rs [hunk 2]
```diff
@@ -104,6 +105,28 @@ impl core::str::FromStr for MemoMessage {
...
+		if msg.token.amount.as_u256() <= ::ibc::bigint::U256::from(10_u64.pow(12)) {
```
Comment (blasrodri) [inline code/parachain/runtime/picasso/src/ibc.rs:123]: where's this limit coming from?

Comment (dzmitry-lahoda) [reply code/parachain/runtime/picasso/src/ibc.rs:123]: 1 DOT/PICA

Comment (dzmitry-lahoda) [reply code/parachain/runtime/picasso/src/ibc.rs:123]: 12 decimals.

sure, will need to implement real stuff when have some full design. or split constant per channel (needs decimals, better not to do until assets merged, so in theory can do).

Comment (dzmitry-lahoda) [reply code/parachain/runtime/picasso/src/ibc.rs:123]: we can change any value. so would prefer we have some RL pallet.

**Review by blasrodri — APPROVED** [review]

**Review by blasrodri — APPROVED** [review]

**Comment by blasrodri** [comment]
great job!!

**Pull request merged**